src/ITensorMPS/autompo/opsum_to_mpo_generic.jl

```diff
@@ -299,8 +299,8 @@ function MPO(os::OpSum, sites::Vector{<:Index}; splitblocks=true, kwargs...)::MP
   return M
 end
 
-function MPO(eltype::Type{<:Number}, os::OpSum, sites::Vector{<:Index}; kwargs...)
-  return NDTensors.convert_scalartype(eltype, MPO(os, sites; kwargs...))
+function MPO(elt::Type{<:Number}, os::OpSum, sites::Vector{<:Index}; kwargs...)
```

Comment:
This was a leftover change since I wantd to use the actual `eltype` function here as a test at one point. I can change it back but it's kind of a trivial change.

Reply:
Fine with me.